docs/security/security.adoc

```diff
@@ -0,0 +1,90 @@
+= Cordova Security - Self Defence Checks
+
+As part of the Aerogear JS SDK module, there is a security self defence check package for cordova projects. This package offers multiple security checks that can be used to analyse the security of the device. This can be useful for apps to detect vulnerablities and react accordingly. For example, if the app detects security issue *X* then do *Y*, where *X* could be no lock screen is set and *Y* is show a system prompt to setup the lock screen.
+
+This package offers:
+
+1. A suite of self-defence checks to detect a compromised environment.
+2. Integration with the AeroGear Services Metrics SDK to report self-defence checks data.
+3. Interfaces to access the results of self-defence checks and take appropriate action in application code.
+
+== Installation
+
+The `@aerogear/security` module is available via NPM:
+
+[source,javascript]
+--
+npm install @aerogear/security@latest --save
+--
+
+After node module installation via NPM use the Cordova CLI to add `cordova-plugin-aerogear-security`
+
+[source,javascript]
+--
+cordova plugin add cordova-plugin-aerogear-security
+--
+
+This will install the packages needed with their native dependencies from the NPM registry and properly link all native dependencies to the project's cordova Platforms.
+
+== Usage
+
+To begin using the self defence checks in your app, you'll need to import the following `SecurityService`, `SecurityCheckType` and `SecurityCheckResult` from `@aerogear/security`:
+
+[source,javascript]
+--
+import { SecurityService, SecurityCheckType, SecurityCheckResult } from '@aerogear/security'; 
+--
+
+You will also need to retrieve an instance of `SecurityService`:
+
+[source,javascript]
+--
+const securityService = new SecurityService();
+--
+
+=== Provided Self Defence Checks
+
+The following are the self defence checks provided by the SDK.
+
+*Important* A successful `check` means that the environment the application is running in is more secure than otherwise, as opposed to signalling if a certain feature is enabled.
+
+For example, a check for whether the device is Rooted should return `true` when it is *not* rooted, since this would be the more secure condition.
```

Comment:
`a check for whether the device is Rooted should return `true` portrays the check to be called `isRooted()` rather than `notRooted()` like below.